odoo/modules/module.py

```diff
@@ -31,60 +31,45 @@
 _logger = logging.getLogger(__name__)
 
 # addons path as a list
-ad_paths = []
+# ad_paths is a deprecated alias, please use odoo.addons.__path__
+@tools.lazy
+def ad_paths():
+    _logger.warning('"odoo.modules.module.ad_paths" is a deprecated '
+                    'proxy to "odoo.addons.__path__". Please consider '
+                    'using the later as the former is going to be '
```

Comment:
I think it should be _latter_ here.
```suggestion
                    'using the latter as the former is going to be '
```